scripts/run_tests

```diff
@@ -13,6 +13,14 @@ cargo clippy --all-targets --manifest-path=./sdk/rust/Cargo.toml -- --deny=warni
 cargo test --all-targets --manifest-path=./examples/Cargo.toml
 cargo clippy --all-targets --manifest-path=./examples/Cargo.toml -- --deny=warnings
 
+bazel_build_flags+=(
+  '--keep-going'
+)
+
+# Check everything builds (except Asylo-specific targets).
+bazel build "${bazel_build_flags[@]}" -- //oak/...:all -//oak/server/asylo:all
```

Comment:
Can we tag the Asylo targets as "manual" too? Or, better, something even more specific?

Reply:
The Asylo targets looked tricky because there's Bazel cleverness going on (single target in the BUILD file expanding to multiple targets), so it seemed easier to just skip that directory rather than try to tag the ~10 targets in there.  

The `"manual"` tag appears to be [a special case for Bazel](https://docs.bazel.build/versions/master/guide.html#target-patterns) so I don't think a different tag would work either.